docs/notebooks/dispersion/two_fluid_dispersion_solver.ipynb

```diff
@@ -0,0 +1,271 @@
+{
```

Comment:
<p><strong>Item 1:</strong></p><p> </p><p><code>k = np.linspace(10<strong>-7, 10</strong>-2, int(1E4)) * u.m ** -1</code></p><p> </p><p><code>num</code> for <code>linespace</code> needs to be an integer.</p><p> </p><p><strong>Item 2:</strong></p><p> </p><p>In your parameter list you should define <code>ion = "p+"</code> and pass this into <code>ion_sound_speed</code>, <code>Alfven_speed</code> , and <code>gyrofrequency</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/932/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>